lib/hunter/commands/send.rb

```diff
@@ -66,20 +66,21 @@ def run
 
           request.body = data.to_json
 
-          loop do
-            response = send_request(http, request)
-            if !retry_interval || response&.code == '200'
-              break
-            end
-            sleep(retry_interval.to_f)
+          if send_request(http, request)&.code != '200' && retry_interval
+            begin
+              sleep(retry_interval.to_f)
+            end while send_request(http, request)&.code != '200'
           end
         end
       end
 
       private
 
       def retry_interval
-        Config.retry_interval || @options.retry_interval
+        @retry_interval ||= begin
+          ri = Config.retry_interval || @options.retry_interval
+          ri.nil? nil : max(5.0, ri.to_f)
```

Comment:
`max` isn't a method; you may be looking for `Array#max` (`[5.0, ri.to_f].max`).

There is also a missing `?` here to define the ternary operator:

```diff
diff --git a/lib/hunter/commands/send.rb b/lib/hunter/commands/send.rb
index 5f2a469..95ec6be 100644
--- a/lib/hunter/commands/send.rb
+++ b/lib/hunter/commands/send.rb
@@ -79,7 +79,7 @@ module Hunter
       def retry_interval
         @retry_interval ||= begin
           ri = Config.retry_interval || @options.retry_interval
-          ri.nil? nil : max(5.0, ri.to_f)
+          ri.nil? ? nil : [5.0, ri.to_f].max
         end
       end
```